library/forecastwrapper.py

```diff
@@ -198,10 +208,48 @@ def get_weather_ts(self, date):
         avg_temp = self._get_daily_avg(forecast = forecast, key = 'temperature')
         day_data.update({'temperature':avg_temp})
 
+        #add daytime Cloud Cover
+        if self.daytimeCloudCover:
+            dtcc = self._get_daytime_avg(forecast = forecast, key= 'cloudCover')
+            day_data.update({'daytimeCloudCover':dtcc})
+        #add daytime Temperature
+        if self.daytimeTemperature:
+            dtt = self._get_daytime_avg(forecast = forecast, key='temperature')
+            day_data.update({'daytimeTemperature':dtt})
+
         #convert to a single row in a dataframe and return
         daylist = [pd.Series(day_data)]
         return pd.concat(daylist,axis=1).T
 
+    def _get_daytime_avg(self, forecast, key):
+        """
+        Calculate the average for a given value during daytime hours (from sunrise to sunset)
+        :param forecast: Forecast object
```

Comment:
It's done automatically by Pycharm…

On Thu, Oct 29, 2015 at 9:49 AM, saroele <notifications@github.com> wrote:

>>          #convert to a single row in a dataframe and return
>>          daylist = [pd.Series(day_data)]
>>          return pd.concat(daylist,axis=1).T
>>  
>> +    def _get_daytime_avg(self, forecast, key):
>> +        """
>> +        Calculate the average for a given value during daytime hours (from sunrise to sunset)
>> +        :param forecast: Forecast object
> @JrtPec  looks like you change the docstring formatting here.  Why?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/opengridcc/opengrid/pull/72/files#r43361228